requirements.txt

```diff
@@ -14,7 +14,7 @@ nbsphinx
 pygments>=2.4.1
 # Developers tools - code formatting
 cpplint
-black
+black==20.8b1 
```

Comment:
While locking black version is a very good idea, I would rather not lock ourself to a beta version =)

Reply:
black is still in a beta mode so all release have a b in their tag. Or I don't set the version and we get always the latest ?

Reply:
My bad, this works then